src/patients/patient-slice.ts

```diff
@@ -81,9 +81,9 @@ export const updatePatient = (patient: Patient, history: any): AppThunk => async
   Toast(
     'success',
     il8n.t('Success!'),
-    `${il8n.t('patients.successfullyUpdated')} ${patient.givenName} ${patient.familyName} ${
-      patient.suffix
-    }`,
+    `${il8n.t('patients.successfullyUpdated')} ${patient.givenName} ${
+      patient.familyName ? patient.familyName : ''
+    } ${patient.suffix ? patient.suffix : ''}`.trimEnd(),
```

Comment:
I think rather than building the name we could use the full name property by doing `patient.fullName`